oracle/feed_stork.go

```diff
@@ -134,10 +134,14 @@ func (f *storkPriceFeed) PullPrice(_ context.Context) (
 }
 
 // ConvertDataToAssetPair converts data get from websocket to list of asset pairs
-func ConvertDataToAssetPair(data Data, assetId string) (result oracletypes.AssetPair) {
+func ConvertDataToAssetPair(data Data, assetId string, refTimestamp uint64) (result oracletypes.AssetPair) {
 	var signedPricesOfAssetPair []*oracletypes.SignedPriceOfAssetPair
 	for i := range data.SignedPrices {
 		newSignedPriceAssetPair := ConvertSignedPrice(data.SignedPrices[i])
+		if ConvertTimestampToSecond(refTimestamp) != newSignedPriceAssetPair.Timestamp {
```

Comment:
I'd add a comment to make it clear
```suggestion
		// newSignedPriceAssetPair.Timestamp is in seconds
		if ConvertTimestampToSecond(refTimestamp) != newSignedPriceAssetPair.Timestamp {
```